README.md

```diff
@@ -13,7 +13,8 @@ LumApps SDK is a set of tools to manipulate the [LumApps API](https://api.lumapp
 
 This includes:
 
-- A client that support all the routes of the API (located in folder lumapps)
+- A BaseClient that support all the routes of the API (located in folder lumapps)
```

Comment:
I would prefer something like:
"A client (BaseClient) that supports...
A client (LumAppsClient) that provides..."